cogs/ReactForRole.py

```diff
@@ -217,6 +271,160 @@ async def rfr_edit_title(self, ctx: commands.Context):
+        if not image or image == "":
+            await ctx.send("Okay, cancelling command.")
+        if isinstance(image, discord.Attachment):
+            # correct type
+            if not image.url:
+                KoalaBot.logger.error(f"Attachment url not found, details : {image}")
+                raise commands.BadArgument("Couldn't get an image from the message you sent.")
+            else:
+                embed.set_thumbnail(url=str(image.url))
+                await msg.edit(embed=embed)
+                await ctx.send("Okay, set the thumbnail of the thumbnail to your desired image. This will error if you "
+                               "delete the message you sent with the image, so make sure you don't.")
+        elif isinstance(image, str):
+            # no attachment in message, just a raw URL in content
+            img_url = await self.get_image_from_url(ctx, image)
+            embed.set_thumbnail(url=img_url)
+            await msg.edit(embed=embed)
+            await ctx.send("Okay, set the thumbnail of the thumbnail to your desired image.")
+        else:
+            raise commands.BadArgument("Couldn't get an image from the message you sent.")
+
+    @commands.check(KoalaBot.is_admin)
+    @commands.check(rfr_is_enabled)
+    @edit_group.command(name="inline")
+    async def rfr_edit_inline(self, ctx: commands.Context):
+        """
+        Edit the inline property of embed fields in rfr embeds. Can edit all rfr messages in a server or a specific one.
+        User is prompted for whether they'd like inline fields or not, as well as details of the specific message if
+        that option is selected. User requires admin perms
+        :param ctx: Context of the command
+        :return:
+        """
+        await ctx.send("Okay, this will change how your embeds look. By default fields are not inline. However, you can"
+                       " choose to change this for a specific message or all rfr messages on the server. To do this, I'"
+                       "ll need some information though. Can you say if you want a specific message edited, or all mess"
+                       "ages on the server?")
+        input = await self.prompt_for_input(ctx, "all or specific")
+        if not isinstance(input, str) or not input:
+            await ctx.send("Okay, cancelling command")
+        else:
+            input_comm = input.lstrip().rstrip().lower()
+            if input_comm not in ["all", "specific"]:
+                await ctx.send("Okay, cancelling command.")
+            elif input_comm.lstrip().rstrip().lower() == "all":
+                await ctx.send(
+                    "Okay, do you want all rfr messages in this server to have inline fields or not? Y for yes, "
+                    "N for no.")
+                change_all = await self.prompt_for_input(ctx, "Y/N")
+                if not change_all or change_all.rstrip().lstrip().upper() not in ["Y", "N"]:
```

Comment:
This should be something like:
```suggestion
                if not change_all:
                    await ctx.send("Okay, cancelling command")
                    return
                change_all = change_all.rstrip().lstrip().upper()
                if change_all not in ["Y","N"]:
```

Reply:
9330f3a4170ddc7b3228a8dadf9beed9f9442429 fixes